extender.go

```diff
@@ -302,7 +299,7 @@ func (e *Extender) extend(kind string, baseImage v1.Image, logger log.Logger) (v
 	}
 	buildOptions := e.extendOptions()
 	var userID, origUserID, prevUserID, groupID string
-	prevUserID = e.CnbUID // placeholder initial value
+	prevUserID = "original user ID" // placeholder initial value
```

Comment:
If you move `userID, groupID = userFrom(*configFile)` out of the `for` loop you will always know the previous user ID

Reply:
cool - thanks.   actually it may have been an error to have it where it was before?   it's weird because we update `baseImage` and `configFile` in the lower half of the loop but we were setting the userID in the upper half of the loop, so i think that meant it was never getting updated the last time? but idk the unit tests pass either way so maybe i'm missing something.